index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <h1 id="title"></h1> 
```

Comment:
please add a heading in your code

Reply:
Kindly add an explanation of why you requested the review. Remember you are trying to correct here and you want the person you are correcting to understand why you made a particular request.